### PR TITLE
Fix merge issues with jellyfish_fog_aws

### PR DESCRIPTION
**Pull request by calebthompson**
* Re-add missing Infrastructure spec
* Add missing data from database products table

### CODE REVIEW

File: app/models/setting.rb [hunk 1]
```diff
@@ -19,21 +19,23 @@ def to_param
...
-  end
-
   # Returns a hash of setting_fields with their hids as the key and their value as the value
   # check_box fields are considered booleans and anything but 'true' is false
   def settings_hash
     setting_fields.map do |setting|
       value = case setting.field_type
               when 'check_box'
-                'true' == setting.value ? true : false
+                'true' == setting.value
```
Comment (stackus) [inline app/models/setting.rb]: Why is the transform to a boolean being removed?

Comment (calebthompson) [inline app/models/setting.rb]: It's not. `#==` returns a boolean here, so the ternary was unnecessary.

Comment (stackus) [inline app/models/setting.rb]: :facepalm:

Sorry, so it is. I only saw the red removed bits.

File: lib/jellyfish_fog_aws/spec/dummy/app/models/provisioner.rb [hunk 2]
```diff
@@ -1 +1,9 @@
-class Provisioner; end
+class Provisioner
+  attr_reader :order_item
+
+  def initialize(order_item)
+    @order_item = order_item
+  end
+
+  def aws_settings; end
```
Comment (brittballard) [inline lib/jellyfish_fog_aws/spec/dummy/app/models/provisioner.rb]: I'm surprised this is the preferred style. I've always undone this when I stumble across it, I guess I should stop that.

**Comment by brittballard** [comment]
Pretty straight forward. LGTM.

**Pull request merged**